### PR TITLE
[Stats Refresh] Add Insights > All Time Stats

### LINKED ISSUE #10466: [Stats Refresh] Add Insights > All Time Stats
On Stats > Insights, add All Time Stats.
- [ ] With data for Posts, Views, Visitors, Best Views Ever
- [ ] Hide empty rows
- [ ] No data

Design:
all_time_stats

(8)

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #10466 

To test:

---
On a site that has stats for Posts, Views, Visitors, and Best Views Ever, verify all four appear in All Time Stats:

all

---
On a site that does _not_ have all four stats, verify only those that do appear in All Time Stats:

partial

---
On a site that has no All Time Stats, verify the `No data yet` view is displayed:

none

### CODE REVIEW

**Comment by SylvesterWilmott** [comment]
Hey! Looks good. Does the text string also have 16pt bottom margin as well? Seems smaller

**Comment by ScoutHarris** [comment]
Hey @SylvesterWilmott .

Which text string? 

If you're talking about the Latest Post Summary, I haven't done that yet.

If you're talking about the All Time Stats rows, they are 11 points.

card_row

**Comment by ScoutHarris** [comment]
Hey @jklausa . I noticed when you pull to refresh, the All Times Stats rows get duplicated. I thought I fixed that. Apparently I re-broke it. If it's OK with you, I'd prefer to fix it in the next PR to avoid merge conflicts.

**Comment by ScoutHarris** [comment]
And I failed to format the numbers. Hence https://github.com/wordpress-mobile/WordPress-iOS/issues/10501 is born.

**Comment by jklausa** [comment]
>Hey @jklausa . I noticed when you pull to refresh, the All Times Stats rows get duplicated. I thought I fixed that. Apparently I re-broke it. If it's OK with you, I'd prefer to fix it in the next PR to avoid merge conflicts. Which will be coming up shortly.
 
this is one of my favorite bugs ever, i legit just spent five minutes seeing how long can it grow :D

File: WordPress/Classes/Stores/StatsInsightsStore.swift [hunk 1]
```diff
@@ -4,6 +4,7 @@ import WordPressComStatsiOS
 
 enum InsightAction: Action {
     case receivedLatestPostSummary(_ latestPostSummary: StatsLatestPostSummary?)
+    case receivedAllTimeStats(_ allTimeStats: StatsAllTime?)
```
Comment (jklausa) [inline WordPress/Classes/Stores/StatsInsightsStore.swift]: github wat u doin

**Pull request merged**